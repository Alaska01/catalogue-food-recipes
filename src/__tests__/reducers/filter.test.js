import filter from '../../reducers/filter';

describe('filter reducer', () => {
  it('should return initial state', () => {
    expect(filter(undefined, 'All')).toMatchSnapshot();
  });

  it('should handle CHANGE FILTER', () => {
    expect(filter('All', {
      type: 'CHANGE FILTER',
    })).toMatchSnapshot();
  });
});
