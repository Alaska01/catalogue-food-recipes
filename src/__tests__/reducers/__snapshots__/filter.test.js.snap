// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`filter reducer should handle CHANGE FILTER 1`] = `undefined`;

exports[`filter reducer should return initial state 1`] = `"All"`;
